CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Added new config setting positPublisher.defaultConnectServer to pre-populate the Server URL. Also checks `/etc/rstudio/rsession.conf` for `default-rsconnect-server`. (rstudio-pro#7466)
```

Comment:
```suggestion
- Added new config setting `positPublisher.defaultConnectServer` to pre-populate the Server URL. Also checks `/etc/rstudio/rsession.conf` for `default-rsconnect-server`. (rstudio-pro#7466)
```

Also, maybe for the second sentence in that note, is it worth saying that that is for configuration on Posit Workbench? Maybe even just say it automatically picks it up from configuration on Posit Workbench (rather than naming the option and file), and link to the Workbench docs about setting it?

Reply:
Thanks, nice catch on that.